tools/wpt/wpt.py

```diff
@@ -132,7 +132,11 @@ def create_complete_parser():
     for command in commands:
         props = commands[command]
 
-        venv.install_requirements(*props.get("requirements", []))
+        try:
+            venv.install_requirements(*props.get("requirements", []))
+        except Exception:
+            # this may happen if, e.g., a package is only installable on certain platforms
+            continue
```

Comment:
cc @jgraham @Ms2ger 

I'm not sure this is necessarily something that makes sense; I could make all the requirements in `requirements_macos_color_profile.txt` `sys_platform == 'darwin'`, but then we just hit the same problem at import time just below.

Unless we go through and make all the PyObjC imports lazy behind a check that we're running on macOS?

Reply:
Letting this fail would probably be fine. The more complete solution would be allowing the command definition to specify that the command itself should fail if you aren't on macOS.

Reply:
Oh, sorry I missed that this was `create_complete_parser`. I think all it means is we don't generate documentation for this one command, right? So it's not great, but not totally awful either. Should probably still log a warning though.